src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -169,224 +176,668 @@ private sealed class ColInfoEx
             public readonly int NumBurninIter;
             public readonly bool ResetRandomGenerator;
 
-            public ColInfoEx(IExceptionContext ectx, Column item, Arguments args)
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            /// <param name="input">Name of input column.</param>
+            /// <param name="output">Name of output column.</param>
+            /// <param name="numTopic">The number of topics in the LDA.</param>
```

Comment:
The output column should be allowed to be null, and you should update the doc so that it describes that when it is null it replaces the input column.

Reply:
fixed

---
In reply to: [231642951](https://github.com/dotnet/machinelearning/pull/1410#discussion_r231642951) [](ancestors = 231642951)